build_registry.sh

```diff
@@ -28,7 +28,7 @@ BASE_DIR=$(dirname $0)
 . ${BASE_DIR}/setenv.sh
 
 # Build the index server base image
-. ${BASE_DIR}/index/server/build.sh
+ENABLE_HTTP2="false" . ${BASE_DIR}/index/server/build.sh
```

Comment:
We should also accept optional http/2 enablement for the test registry build script as well.